docs/02-guides/maintenance-mode/_category_.yml

```diff
@@ -0,0 +1,7 @@
+label: Maintenance Mode
+collapsible: true
+link:
+  type: generated-index
+  description:
+    Learn how to set your store in maintenance mode and display a maintenance
+    page or use systematic health checks to set maintenance mode automatically.
```

Comment:
**nitpick:** as a reader, the term `systematic` isn't clear to me

**suggestion:** removing it seems to be ok
```suggestion
    page or use health checks to set maintenance mode automatically.
```